components/folder/NavBox.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from "react";
+import { getFolderNavInfo } from "@/api/folder";
+import * as S from "./NavBox.styled";
+import useModal from "@/hooks/useModal";
+import NavItem from "./NavItem";
+import add from "@/public/image/icon/add.svg";
+import addMobile from "@/public/image/icon/add-mobile.svg";
+import shareIcon from "@/public/image/icon/share.svg";
+import penIcon from "@/public/image/icon/pen.svg";
+import deleteIcon from "@/public/image/icon/delete.svg";
+import { ModalParam } from "@/types/Modal";
+import { INavItem } from "@/types/FolderNav";
+import Image from "next/image";
+
+interface FolderNav {
+  navId?: number;
+  onClickNavItem: (navId: number) => void;
+}
+
+const NavBox = ({ navId, onClickNavItem }: FolderNav) => {
+  const { openModal } = useModal();
+  const [navList, setNavList] = useState<INavItem[]>([]);
+  const [currentNav, setCurrentNav] = useState("전체");
+
+  const handleLoadInfo = async () => {
+    const folderNavInfo = await getFolderNavInfo();
+
+    if (folderNavInfo !== null) {
+      setNavList(folderNavInfo.data);
+    }
+  };
+
+  useEffect(() => {
+    handleLoadInfo();
+  }, []);
+
+  useEffect(() => {
+    if (navId === 9999) {
+      setCurrentNav("전체");
+    }
+
+    navList.forEach((navListItem: INavItem) => {
+      if (navListItem.name && navListItem.id === navId)
+        setCurrentNav(navListItem.name);
+    });
+  }, [navId, navList]);
+
+  const handleOpenModal = ({ type, props }: ModalParam) => {
+    openModal({ type, props });
+  };
+
+  return (
+    <>
+      <S.NavWrapBox>
```

Comment:
fragment 안 써도 될 것 같네요??

```suggestion
    <S.NavWrapBox>
```